src/flux/task-step.coffee

```diff
@@ -5,13 +5,17 @@ flux = require 'flux-react'
 
 TaskStepConfig =
 
+  _loaded: (obj, id) ->
+    _.extend(@_local[id], obj)
```

Comment:
Why extend the existing object? If it's just to move the `task_id` then maybe make that explicit?